pkg/features/embedded/elemental-rootfs/usr/lib/systemd/system/elemental-rootfs.service

```diff
@@ -2,7 +2,7 @@
 Description=Elemental system rootfs overlay mounts
 DefaultDependencies=no
 After=initrd-root-fs.target
-Requires=initrd-root-fs.target
+Wants=initrd-root-fs.target
```

Comment:
Relaxing this requirement so we could consider removing the `RemainAfterExit` if needed, which could be useful to track start and stop of the service.